pkg/localize/locales/en/cmd/acl.en.toml

```
@@ -314,7 +315,7 @@ one = 'Deleting ACLs from Kafka instance "{{.Name}}"'
 one = 'Grant an account permissions to create and delete ACLs in the Kafka instance'
 
 [kafka.acl.grantAdmin.cmd.longDescription]
-one = 'This command grants a specified account permission to create and delete ACLs in a Kafka instance.'
+one = 'Grant admin rights to a user. Admin rights enable the user to create and delete Access Control Lists (ACLs) in a Kafka instance.'
```

Not 100% sure about "admin" rights
There are no typical "admin" rights in Kafka, but I think we mean here an "administrative" rights to cluster (AKA alter permission https://docs.confluent.io/platform/current/kafka/authorization.html). 

It think it is better to use full "administrative" rights as "admin" might be missleading for user thinking that they get admin rights (where all they get is to Cluster Alter right)

```suggestion
one = 'Grant administrative rights to a user. Admin rights enable the user to create and delete Access Control Lists (ACLs) in a Kafka instance.'
```